### ISSUE TITLE
[ICC] Professor Putricide engages p2 after pull

### ISSUE THREAD

**Issue by Derolana** [dalaranwow/dalaran-wow#5220]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After pulling PP he somehow decided to skip P1 
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
https://cdn.discordapp.com/attachments/298148740185849856/388068564566802432/WoWScrnShot_120617_222622.jpg
https://streamable.com/xawmg
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Derolana,
Steps of how to reproduce this issue will be great. Did you do something specific, did it happen only once?

**Comment by Derolana** [comment]
only once :) i think we had 1 wipe before it, but I dont know how to reproduce it :( . Yesterday the gm of Pure Blood told me that happened once in his icc25 run.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for this report. I have done many tests and couldn't reproduce this issue either. It looks like it is something very isolated, I will add some logs for it.
